powerlevel9k.zsh-theme

```diff
@@ -1844,7 +1844,7 @@ function _p9k_vcs_gitstatus() {
         esac
       done
       typeset -gFH _P9K_GITSTATUS_START_TIME=$EPOCHREALTIME
-      gitstatus_query -t $timeout -c _p9k_vcs_resume POWERLEVEL9K || return 1
+      gitstatus_query ${GIT_DIR:+-d $GIT_DIR} -t $timeout -c _p9k_vcs_resume POWERLEVEL9K || return 1
```

Comment:
I like it. Let's make it a bit more explicit, like this:

```zsh
  -d ${GIT_DIR:-$PWD}
```

Reply:
Sure. I wasn't sure if always explicitly setting that parameter would mess anything else up. Change coming right up.